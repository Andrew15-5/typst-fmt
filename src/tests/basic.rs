use super::*;
make_test!(unchanged, " ");
make_test!(one_space, "  ");
make_test!(one_space_b, "   ");
make_test!(two_line_max, "\n\n\n");
make_test!(let_stmt_unchanged, "#let ident = variable");
make_test!(let_stmt_period_terminated, "#let ident = variable;");
make_test!(let_stmt_no_spacing, "#let ident=variable");
make_test!(
    content_block_spaced,
    "#[\n_Glaciers_ form an important part \nof the earth's climate system.]"
);
// todo, fix array formatting trailing comma
make_test!(parenthesized, "#(auto,)");
