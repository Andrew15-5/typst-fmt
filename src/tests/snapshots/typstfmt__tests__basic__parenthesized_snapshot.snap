---
source: src/tests/basic.rs
description: "INPUT\n===\n\"#(auto,)\"\n===\n#(auto,)\n===\nFORMATTED\n===\n#(auto,)"
expression: formatted
---
"#(auto,)"
